onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -120,7 +115,9 @@ class ConvActivationSelector : public NodeSelector {
     }
 
     // check EP type and activation
-    if (node_ep == kCudaExecutionProvider || node_ep == kRocmExecutionProvider) {
+    if (node_ep == kCudaExecutionProvider) {
```

Comment:
You may disable the fusion when prefer_nhwc provider option is True. Otherwise, keep it.

Reply:
It will not be eligible for NHWC CUDA anyway.

Reply:
Same question - Any reason to not keep legacy fusion option when NHWC is not enabled ?

Reply:
We will enable prefer NHWC by default in 1.20 release.

We can have more discussion later for cudnn 8 support  in 1.20 release. If it is essential, we can still bring back the fusion if necessary later.